extensions/amp-accordion/1.0/accordion.js

```diff
@@ -177,9 +261,9 @@ export function AccordionSection({
 
   useLayoutEffect(() => {
     if (registerSection) {
-      return registerSection(id, defaultExpanded);
+      return registerSection(id, defaultExpanded, propId);
```

Comment:
Can we just use real ID instead of auto-gen ID when specified?

Reply:
Updated in latest revision.  Per offline conversation we will use real ID or auto-gen ID here.  This potentially allows user to access sections without explicit ID defined, but we will assume user will use component correctly or get undefined behavior.